python/ray/serve/tests/test_batching.py

```diff
@@ -214,6 +215,36 @@ async def call(arg):
         await t3
 
 
+@pytest.mark.asyncio
+async def test_batch_timeout_empty_queue():
+    """Check that Serve waits for before creating the first batch."""
```

Comment:
specify that it should wait for the timeout _after_ the first request for a new batch arrives rather than after the first batch ends

Reply:
Sounds good, I reworded the docstring.